deploy/008-deploy-prime.ts

```diff
@@ -0,0 +1,128 @@
+import { ethers } from "hardhat";
+import { Address, DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { Contracts as Mainnet } from "../networks/mainnet.json";
+import { Contracts as Testnet } from "../networks/testnet.json";
+
+interface AddressConfig {
+  [key: string]: {
+    [key: string]: Address;
+  };
+}
+
+const ADDRESSES: AddressConfig = {
+  bsctestnet: Testnet,
+  bscmainnet: Mainnet,
+};
+
+const OTHER_ADDRESSES: any = {
+  bsctestnet: {
+    acm: "0x45f8a08F534f34A97187626E05d4b6648Eeaa9AA",
+    psr: "0x25c7c7D6Bf710949fD7f03364E9BA19a1b3c10E3",
+    oracle: "0x3cD69251D04A28d887Ac14cbe2E14c52F3D57823",
+  },
+  bscmainnet: {
+    acm: "0x4788629ABc6cFCA10F9f969efdEAa1cF70c23555",
+    psr: "0x3DA3619EE1FE1031051c3d0dfFe252a145F2630D",
+    oracle: "0x6592b5DE802159F3E74B2486b091D11a8256ab8A",
+  },
+};
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, network, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  const networkName = network.name === "bscmainnet" ? "bscmainnet" : "bsctestnet";
+  const stakingPeriod = networkName === "bscmainnet" ? 90 * 24 * 60 * 60 : 60 * 10;
+  const maximumXVSCap =
+    networkName === "bscmainnet" ? ethers.utils.parseEther("100000") : ethers.utils.parseEther("100");
+  const minimumXVS = networkName === "bscmainnet" ? ethers.utils.parseEther("1000") : ethers.utils.parseEther("10");
```

Comment:
I think we can use the same values for these variables, in testnet and mainnet, and it could create less confusion testing it